tests/test_schunk_fill.py

```diff
@@ -0,0 +1,65 @@
+#######################################################################
```

Comment:
As this test builds a `SChunk`, I'd move it to `test_schunk_constructor.py`

Reply:
Fixed in 296712f60e1f58169fe0f3d0766d30ff8d981e30.